README.md

```diff
@@ -1,2 +1,40 @@
-# AtlasPack
-Combined cortical/subcortical atlases for xcp-d and qsiprep
+# Combining cortical and subcortical atlases for use in XCPD and QSIPrep
+
+There are many popular brain atlases, but they're all in different versions of MNI space. They also cover 
+different structures. We want to get them all into one MNI version correctly - using the templateflow 
+framework - so they can be used to make structural and functional connectivity matrices.
+
+## Things to worry about
+
+We ultimately want square connectivity matrices with exactly the same regions represented in the rows/columns to come out of xcpd 
+and qsiprep. Here are some of the problems we will run into:
+
+ 1. Some regions are so small that when they're mapped to a low-res BOLD/DWI volume they will disappear. Should we allow this? 
+    In some cases they are from dividing a region into subcomponents. We could recombine them.
```

Comment:
I wonder what nilearn's behavior is? I assume it just returns NaNs for the missing regions. I'd be fine with NaNs if the region disappears or is entirely outside the brain mask, generally speaking.

Reply:
I'm mostly worried about what this means for those interpreting the connectivity matrices without thinking too hard about it. A column of NaNs isn't a big deal software-wise, but if someone is doing network science stuff on these matrices it's a big deal to have a fully disconnected node. We also know that there should almost never be an actually disconnected node in the brains we'll be analyzing. We looked at this problem [here](https://direct.mit.edu/netn/article/1/4/446/5397) and it still haunts me